src/installer/wheels.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+import csv
+import dataclasses
+import pathlib
+import warnings
+
+from typing import Iterator, Optional
+
+
+class SuperfulousRecordColumnsWarning(UserWarning):
+    pass
+
+
+@dataclasses.dataclass()
+class Hash:
+    name: str
+    value: str
+
+    @classmethod
+    def parse(cls, s: str) -> Optional[Hash]:
+        name, value = s.split("=", 1)
+        return cls(name, value)
+
+
+@dataclasses.dataclass()
+class Record:
+    path: pathlib.PurePosixPath
+    hash_: Optional[Hash]
```

Comment:
Does this need to be its own dataclass, or could it just be a tuple?

Reply:
I think it will be a `collections.namedtuple()` eventually (considering the plan seems to be to support Python 2)

Reply:
Thinking about this a bit more, maybe Hash really needs to be a class, so we can put a `validate(data: bytes) -> bool` on it.